content/docs/1.6.0/advanced-resources/deploy/storage-network.md

```diff
@@ -14,7 +14,7 @@ Longhorn will apply the same annotation to any new instance-manager, backing-ima
 
 > **Warning**: Do not change this setting with volumes attached.
 >
-> Longhorn will try to block this setting update when there are attached volumes.
+> Longhorn will try to restart all instance-manager, and backing-image-manager pods if all volumes are detached and eventually restart the instance manager pod without instances running on the instance manager..
```

Comment:
```suggestion
> When all volumes are detached, Longhorn attempts to restart all Instance Manager and Backing Image Manager pods.
```

Reply:
> eventually restart the instance manager pod without instances running on the instance manager.

This is unclear. Do you mean that the pod is restarted only when no instances are running on the Instance Manager?

Reply:
Yes, it is.
The instance manager pod will be restarted only when no instances are running on the Instance Manager and the setting is not applied to the pod.

Reply:
Thanks. I have updated the suggestion.

Reply:
Modified. Thanks.